sources/models/contacts.js

```diff
@@ -1,10 +1,21 @@
+const dateToStr = webix.Date.dateToStr("%Y-%m-%d");
 export const contacts = new webix.DataCollection(
 	{
 		url: "http://localhost:8096/api/v1/contacts/",
 		save: "rest->http://localhost:8096/api/v1/contacts/",
 		scheme: {
 			$init: (obj) => {
 				obj.value = `${obj.FirstName}  ${obj.LastName}`;
+				obj.StartDate = webix.i18n.parseFormatDate(obj.StartDate);
+				obj.Birthday = webix.i18n.parseFormatDate(obj.Birthday);
+			},
+			$change: (obj) => {
+				obj.StartDate = webix.i18n.parseFormatDate(obj.StartDate);
+				obj.Birthday = webix.i18n.parseFormatDate(obj.Birthday);
```

Comment:
Seems not necessary as you do this on init and treat them as Date objects 
By the way, the same is true fo Activities model, as the DueDate is always a Date on the client-side.

Reply:
fixed